Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Go Web Application\n\nThis is a simple website written in Golang. It uses the `net/http` package to serve HTTP requests.\n\n## Running the server\n\nTo run the server, execute the following command:\n\n```bash\ngo run main.go\n```\n\nThe server will start on port 8080. You can access it by navigating to `http://localhost:8080/courses` in your web browser.\n\n## Looks like this\n\n![Website](static/images/golang-website.png)\n\n\n# go-web-app\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision c423f658e2d397660e0710452c13558478e36492)
+++ b/README.md	(date 1724014589730)
@@ -18,3 +18,4 @@
 
 
 # go-web-app
+# go-web-app
Index: Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile	(revision c423f658e2d397660e0710452c13558478e36492)
+++ b/Dockerfile	(date 1724013862282)
@@ -1,0 +1,22 @@
+FROM golang:1.22.5 as build
+
+WORKDIR /app
+
+COPY go.mod .
+
+RUN go mod download
+
+COPY . .
+
+RUN go build -o main .
+
+# Final Stage - Using Distroless Image
+FROM gcr.io/distroless/base
+
+COPY --from=build /app/main .
+
+COPY --from=build /app/static ./static
+
+EXPOSE 8080
+
+CMD ["./main"]
\ No newline at end of file
